scripts/extract_pdf_text.py

```diff
@@ -0,0 +1,22 @@
+import fitz
```

Comment:
so strange for them to keep this name around...

Reply:
agreed